components/infobox/mock/infobox_mock_league.lua

```diff
@@ -0,0 +1,23 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Mock/League
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local League = Lua.import('Module:Infobox/League', {requireDevIfEnabled = true})
+
+local mockTournament = {}
+
+function mockTournament.setUp(data)
+	League:_definePageVariables(data)
+end
+
+function mockTournament.tearDown()
+	League:_definePageVariables({})
```

Comment:
will not reset `tournament_parent` var due it having a fallback of `mw.title.getCurrentTitle().prefixedText`